tests/system.rs

```diff
@@ -167,23 +168,28 @@ fn get_config(config_file: Option<&str>) -> Box<Config> {
 // Reads significant comments of the form: // rustfmt-key: value
 // into a hash map.
 fn read_significant_comments(file_name: &str) -> HashMap<String, String> {
-    let file = fs::File::open(file_name).ok().expect(&format!("Couldn't read file {}.", file_name));
+    let file = fs::File::open(file_name)
+                   .ok()
+                   .expect(&format!("Couldn\'t read file {}.", file_name));
     let reader = BufReader::new(file);
     let pattern = r"^\s*//\s*rustfmt-([^:]+):\s*(\S+)";
     let regex = regex::Regex::new(&pattern).ok().expect("Failed creating pattern 1.");
 
     // Matches lines containing significant comments or whitespace.
     let line_regex = regex::Regex::new(r"(^\s*$)|(^\s*//\s*rustfmt-[^:]+:\s*\S+)")
-        .ok().expect("Failed creating pattern 2.");
+                         .ok()
+                         .expect("Failed creating pattern 2.");
 
     reader.lines()
           .map(|line| line.ok().expect("Failed getting line."))
           .take_while(|line| line_regex.is_match(&line))
           .filter_map(|line| {
-              regex.captures_iter(&line).next().map(|capture| {
-                  (capture.at(1).expect("Couldn't unwrap capture.").to_owned(),
-                   capture.at(2).expect("Couldn't unwrap capture.").to_owned())
-              })
+              regex.captures_iter(&line)
+                   .next()
+                   .map(|capture| {
```

Comment:
There's quite a few places where do this pattern of line breaking. I do think we should fix it before landing this PR.